test/drbgtest.c

```diff
@@ -669,6 +676,40 @@ static int test_drbg_reseed(int expect_success,
     return 1;
 }
 
+
+#if defined(OPENSSL_SYS_UNIX)
+/*
+ * Test whether master, public and private DRBG are reseeded after
+ * forking the process.
+ */
+static int test_drbg_reseed_after_fork(RAND_DRBG *master,
+                                       RAND_DRBG *public,
+                                       RAND_DRBG *private)
+{
+    pid_t pid;
+    int status=0;
+
+    pid = fork();
+    if (!TEST_int_ge(pid, 0))
+        return 0;
+
+    if (pid > 0) {
+        /* I'm the parent; wait for the child and check its exit code */
+        return TEST_int_eq(waitpid(pid, &status, 0), pid) && TEST_int_eq(status, 0);
```

Comment:
It is worthwhile handling an `EINTR` as a repeat here?
I can't imagine how it would occur, but one never knows.

Reply:
Hmmm... I'd rather not make the code unnecessary complicated to handle a case that might never occur. If it turns out that the test case fails (sporadically) because the `waitpid()` call gets interrupted, then it can still be done IMHO.